prow/scripts/resources/README.md

```diff
@@ -0,0 +1,18 @@
+## Overview
+
+The folder Contains files which are directly used by prow pipeline scripts.
+
+## Content
+
+```
+├── kube-dns-stub-domains-patch.yaml                          # Enable stubdomain build.kyma-project.io and provide Google root DNS servers IPs.
+├── limitrange-patch.yaml                                     # Increase kyma-system namespace max memory request for containers.
+├── prometheus-operator-stackdriver-patch.yaml                # Inject stackdriver collector sidecar, set metrics filters, enable scraping stackdriver target.
```

Comment:
```suggestion
├── prometheus-operator-stackdriver-patch.yaml                # Injects the Stackdriver collector sidecar, sets metric filters, and enables scraping Stackdriver target.
```

Reply:
Scraping?

Reply:
Scraping is a word used by Prometheus documentation to describe process of collecting metric values from monitored components.